Essay_Analysis_Tool/DiffViewerForm.cs

```diff
@@ -95,33 +95,41 @@ private void FastColoredTextBox_VisibleRangeChanged(object sender, EventArgs e)
                 return;
 
             var vPos = (sender as FastColoredTextBox).VerticalScroll.Value;
+            var hPos = (sender as FastColoredTextBox).HorizontalScroll.Value;
             var curLine = (sender as FastColoredTextBox).Selection.Start.iLine;
 
             if (sender == fctb2)
-                UpdateScroll(fctb1, vPos, curLine);
+                UpdateScroll(fctb1, vPos, hPos, curLine);
             else
-                UpdateScroll(fctb2, vPos, curLine);
+                UpdateScroll(fctb2, vPos, hPos, curLine);
 
             fctb1.Refresh();
             fctb2.Refresh();
```

Comment:
If i remove the two `fctb.Refresh()` method the line number section become less laggy but it updates the two FCTB instance less often so when scrolling the text content jump instead of smoothly move left/right/top/down in the screen.